R/z_animintHelpers.R

```diff
@@ -162,6 +162,33 @@ setUpdateAxes <- function(theme, options_list){
 }
 
 
+#' Get x/y axes font size from params text, axis.text and axis.text.x/y via theme()
+#'
+#' @param xy.size size params defined in axis.text.x or axis.text.y
+#' @param theme.pars theme settings
+#' @return size
+axesTextSize <- function(xy.size, theme.pars){
+  text.size <- if(is.null(theme.pars[["text"]]$size)){11}else{theme.pars[["text"]]$size}
+  axis.text.size <- if(is.null(theme.pars[["axis.text"]]$size)||theme.pars[["axis.text"]]$size==rel(0.8)){
```

Comment:
In the output, the axis.text default to 0.8, and printed out as `0.8 *` because [this line of code](https://github.com/tdhock/animint2/blob/4b1049bc1d8ee8dbd78b31f52a5863496fae5931/R/theme-elements.r#L101) in theme-element.r

Reply:
This function should be modified to be more clear and less fragile.